external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-if(${ENABLE_PCMSolver})
-    find_package(PCMSolver 1.1.10 CONFIG QUIET)
+    if(${ENABLE_PCMSolver})
+    find_package(PCMSolver 1.1.11 CONFIG QUIET)
```

Comment:
@robertodr, is 1.1.11 required or is 1.1.10 still compatible? if the former, all good. if the latter, then switch this and the psi4/CM one back to 10, please, while keeping the GIT_TAG one at 11. that way things won't break unnecessarily. thanks!

Reply:
1.1.10 is still compatible. Fixed according to your comment.